docker/README.md

```diff
@@ -0,0 +1,38 @@
+# Running Bisq using Docker
+
+Running Bisq using Docker has currently only been tested using Linux.
+
+## Build the image
+
+```sh
+docker build -t bisq .
+```
+
+## Running Bisq
```

Comment:
It would be nice to have docker-compose configured with several containers like 
- prod, which would have default volume bound to `$HOME/.local/share/Bisq`
- dev, without data dir volume bound

That way user would only need to call `docker-compose up prod` or `docker-compose up dev`